ygen/testdata/schema/openconfig-options-compress-fakeroot.formatted-txt

```diff
@@ -128,6 +128,20 @@ type Bgp_Neighbor_EnabledAddressFamily_Union_Uint32 struct {
 // implements the Bgp_Neighbor_EnabledAddressFamily_Union interface.
 func (*Bgp_Neighbor_EnabledAddressFamily_Union_Uint32) Is_Bgp_Neighbor_EnabledAddressFamily_Union() {}
 
+// To_Bgp_Neighbor_EnabledAddressFamily_Union takes an input interface{} and attempts to convert it to a struct
+// which implements the Bgp_Neighbor_EnabledAddressFamily_Union union. Returns an error if the interface{} supplied
+// cannot be converted to a type within the union.
+func (t *Bgp_Neighbor) To_Bgp_Neighbor_EnabledAddressFamily_Union(i interface{}) (Bgp_Neighbor_EnabledAddressFamily_Union, error) {
+	switch v := i.(type) {
+	case E_OpenconfigOptions_AFI:
+		return &Bgp_Neighbor_EnabledAddressFamily_Union_E_OpenconfigOptions_AFI{v}, nil
+	case uint32:
+		return &Bgp_Neighbor_EnabledAddressFamily_Union_Uint32{v}, nil
+	default:
+		return nil, fmt.Errorf("cannot convert %!v(MISSING) to Bgp_Neighbor_EnabledAddressFamily_Union, unknown union type, got: %!T(MISSING), want:E_OpenconfigOptions_AFI uint32 ", i, i)
+	}
```

Comment:
Generation problem @MISSING, here and below?

Reply:
Sorry - issue with copy and pasting expected output as I vetted these. Will fix.